submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                    pattern="(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*"
+                    title="Enter your password. Minimum 8 characters, UPPERCASE, lowercase and Number."
+                    minlength="6"
+                    required
+                  /><span class="validation-sign"></span>
+                </div>
+              </div>
+              <div class="payment-method-choose">
+                <div class="payholder">
+                  <label for="card">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="card"
+                      class="input"
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/visa.svg" alt="" class="img-card" /><img
+                        src="./img/mastercard.svg"
+                        alt=""
+                        class="img-card"
+                      />
+                    </div>
+                  </label>
+                  <label for="paypal">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="paypal"
+                      class="input"
+                      checked
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/paypal.svg" alt="" class="img-pay" />
+                    </div>
+                  </label>
+                  <label for="amazon">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="amazon"
+                      class="input"
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/amazon.svg" alt="" class="img-pay" />
+                    </div>
+                  </label>
+                </div>
+              </div>
+              <div class="field-text">
+                <div class="field-container field-card">
+                  <h5 class="above-text">CARD NUMBER</h5>
+                  <input
+                    type="text"
+                    inputmode="numeric"
+                    id="card"
+                    class="text text-card"
+                    name="card"
+                    pattern="\d*"
+                    title="Enter your credit card number. Example: 1234 5678 9012 4567"
+                    minlength="16"
+                    maxlength="16"
+                    required
+                  /><span class="cc-number-img"></span>
+                </div>
+                <div class="field-container field-date">
+                  <h5 class="above-text">EXPIRY DATE</h5>
+                  <input
```

Comment:
You may use HTML5 feature `<input type='date'>` or `<input type='month'>` for this

Reply:
I did not see anything like that before, thanks for advice! However, It seems to me It's better to use my variation in this situation.

Reply:
@Zakkarat  It is up to you. But it is more correct to use the features which are semantically true.